### PR TITLE
Swap text-based on and off toggles to non-text based

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
Our text-based toggles are particularly problematic when translated to other languages.  This is a proposal to use toggles that look like [Patternfly's switches](https://pf4.patternfly.org/components/Switch/examples/).  The non-disabled versions would look like this:

Screen Shot 2019-08-05 at 11 09 15 AM

The disabled versions would look like this:

Screen Shot 2019-08-05 at 11 09 31 AM

Here's an example screenshot of the poor behavior:
53660789-b6866780-3c2c-11e9-9a30-07ab3bfa4c10

A lot of the code removal here has to do with old job submission code.  Some of this old code referenced the `ScheduleToggle` class(es) which is what led me down that path.  This PR should replace _all_ of the old on/off toggles with these new toggles.  Most, if not all, of these toggles appear in lists and forms throughout the application.  They are particularly pervasive in the settings forms and host lists but they also appear in the Job Template form, Notifications list and Survey Maker amongst others.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION
6.1.0

##### ADDITIONAL INFORMATION
Manually tested as many of the toggles as I could reasonably locate.  I also cruised through the UI with a system auditor user to make sure that toggles were properly disabled.  I did find that the survey maker toggle has never been disabled for users that cannot edit the JT so I fixed that.

### CODE REVIEW

**Comment by ryanpetrello** [comment]
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

File: awx/ui/client/src/shared/form-generator.js [hunk 1]
```diff
@@ -512,21 +512,16 @@ angular.module('FormGenerator', [GeneratorHelpers.name, 'Utilities', listGenerat
...
-                    html += "' ";
-                    html += (field.ngDisabled) ? `ng-disabled="${field.ngDisabled}" ` : "";
-                    html += " class='ScheduleToggle-switch is-on' ng-click='" + field.ngClick + "' translate>" + i18n._("ON") + "</button><button type='button' ng-show='!" + form.iterator + "." ;
-                    html += (field.flag) ? field.flag : "enabled";
-                    html += "' ";
-                    html += (field.ngDisabled) ? `ng-disabled="${field.ngDisabled}" ` : "";
-                    html += " class='ScheduleToggle-switch' ng-click='" + field.ngClick + "' translate>" + i18n._("OFF") + "</button></div></div>";
+                    html += `
+                        <div class="Field-header--${key} ${field['class']} ${field.columnClass}">
+                            <span class="awxSwitch-outer" ng-class="{'awxSwitch-on': ${"flag" in field} ? ${form.iterator}.${field.flag} : ${form.iterator}.enabled, 'awxSwitch-disabled': ${"ngDisabled" in field} ? ${field.ngDisabled} : false}" aw-tool-tip="${field.awToolTip}" data-placement="${field.dataPlacement ? field.dataPlacement : 'right'}" data-tip-watch="${field.dataTipWatch}">
```
Comment (jlmitch5) [inline awx/ui/client/src/shared/form-generator.js]: I haven't seen a ternary with an in iterator like this before.  is this an angular thing?  I'm not sure how it works.

Comment (mabashian) [reply awx/ui/client/src/shared/form-generator.js]: Yea you can have a ternary inside of the ng-class directive.  We use this same syntax in a couple of other places.

**Review by jlmitch5 — APPROVED** [review]
looking awesome!  great job on this

**Pull request merged**